### PR TITLE
add community spotlight

### PR DESCRIPTION
**Pull request by xmulligan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ It's best if you use a public communication channel whenever possible; however,
 
 ## Success Stories
 
+[Matei David](https://www.cncf.io/spotlights/matei-david/)
```
Comment (idvoretskyi) [inline README.md:39]: Let's add https://www.cncf.io/spotlights/cncf-maintainer-spotlight-goutham-veeramachaneni/ as well, Goutham is our intern in the past.

File: README.md [hunk 2]
```diff
@@ -36,6 +36,8 @@ It's best if you use a public communication channel whenever possible; however,
...
+
```
Comment (xmulligan) [inline README.md:40]:
```suggestion
[Goutham Veeramachaneni](https://www.cncf.io/spotlights/cncf-maintainer-spotlight-goutham-veeramachaneni/)
```

File: README.md [hunk 3]
```diff
@@ -36,6 +36,8 @@ It's best if you use a public communication channel whenever possible; however,
 
 ## Success Stories
 
+[Matei David](https://www.cncf.io/spotlights/matei-david/)
+[Goutham Veeramachaneni](https://www.cncf.io/spotlights/cncf-maintainer-spotlight-goutham-veeramachaneni/)
```
Comment (xmulligan) [inline README.md:40]:
```suggestion
[Goutham Veeramachaneni](https://www.cncf.io/spotlights/cncf-maintainer-spotlight-goutham-veeramachaneni/)

```

File: README.md (hunk 1 above)
Comment (xmulligan) [reply README.md:39]: thanks, didn't know. Have added now

**Review by idvoretskyi — APPROVED** [review]

**Comment by idvoretskyi** [comment]
Thanks, @xmulligan!

**Pull request merged**